src/WoodburyMatrices.jl

```diff
@@ -88,7 +88,16 @@ end
 _ldiv!(dest, W, A, B) = _ldiv!(dest, W, lu(A), B)
 
 det(W::AbstractWoodbury) = det(W.A)*det(W.C)/det(W.Cp)
-
+logdet(W::AbstractWoodbury) = logdet(W.A) + logdet(W.C) - logdet(W.Cp)
+function logabsdet(W::AbstractWoodbury)
+    lad_A = logabsdet(W.A)
+    lad_C = logabsdet(W.C)
+    lad_Cp = logabsdet(W.Cp)
+    lad = lad_A[1] + lad_C[1] - lad_Cp[1]
+    s = lad_A[2] * lad_C[2] / lad_Cp[2]
+    return lad, s
+end
+end
```

Comment:
The extra `end` here terminates the module

Reply:
whoopsies- copy and paste introduces a bug that takes me 30 minutes to figure out (part 139587138951)